src/server/services/MediaResourceServices.js

```diff
@@ -17,10 +17,14 @@ class MediaResourceServices {
       courseLabel
     );
 
+    const musicresCount = await MediaQuery.getMusicResCountByCourse(
+      courseLabel
```

Comment:
@maxpluez So it seems that once someone is authenticated via LTI they can then, potentially, call our end points for any course label and get the media counts. That seems like a security concern. The endpoint already knows the context.label the user is logged into. Can't we get that data from the LTIjs ContextToken (https://cvmcosta.me/ltijs/#/provider?id=contexttoken)? Why have it passed as a parameter?

Reply:
Wait, this function is only called by the api routes. the api routes call this function with context token.

Reply:
@maxpluez  what line is that call happening?

Reply:
@maxpluez  okay I see it at src/server/api/medias/musicres.js

Still should check role

Reply:
It's in server/api/medias/index.js

Reply:
Oh yeah both of them call it

Reply:
So technically speaking all js files under /services are only in service to the api routes. Users shouldn't be able to directly call them..